roles/deploy/tasks/main.yml

```diff
@@ -196,14 +195,15 @@
     - precompile assets
     - restart unicorn
 
-# tag to trigger a unicorn restart
-- name: reset
-  # needs to do something here
-  command: echo 1
-  tags:
-    - reset
-    - skip_ansible_lint
-  notify:
-    - restart unicorn
```

Comment:
I don't see a replacement for notifying the unicorn handler.

Reply:
There are multiple `notify`s to the same handler in `deploy.yml`. It's still there!